index.html

```diff
@@ -0,0 +1,686 @@
+                            <p class="first-tile-date">Thursday - March 12,2020</p>
+                        </div>
+
+                        <div class="main-tiles-title-container">
+                            <h1 class="main-tiles-title">Researchers use AI to translate text found on ancient clay tablets</h1>
+                            <p class="main-tiles-info">by <a href="#">Thomas Macaulay</a> <i class="far fa-clock"></i> 2 hours ago <i class="fas fa-share-alt"></i> 9</p>
+                        </div>
+
+                    </div>
+
+                </div>
+
+                <div class="main-tiles-second-tile-container">
+
+                        <div class=" main-second-tile-top-background">
+
+                                    <div class="second-tile-gradient">
+                                    </div>
+
+                                    <div class="main-tiles-second-tile">
+                                        <h1 class="main-tiles-second-title">Researchers use AI to translate text found on ancient clay tablets</h1>
+                                        <p class="main-tiles-info">by <a href="#">Thomas Macaulay</a> <i class="far fa-clock"></i> 2 hours ago <i class="fas fa-share-alt"></i> 9</p>
+                                    </div>
+
+                        </div>
+
+                        <div class=" main-second-tile-bottom-background">
+                                    <div class="second-tile-gradient">
+                                    </div>
+
+                                    <div class="main-tiles-second-tile">
+                                        <h1 class="main-tiles-second-title">Researchers use AI to translate text found on ancient clay tablets</h1>
+                                        <p class="main-tiles-info">by <a href="#">Thomas Macaulay</a> <i class="far fa-clock"></i> 2 hours ago <i class="fas fa-share-alt"></i> 9</p>
+                                    </div>
+
+                        </div>
+
+                </div>
+
+        </div>
+
+        <div class="latest-news general-container">
+            <h1 class="section-title"><a  class="section-title-a" href="#">Latest News</a></h1>
+            <div class="latest-news-items-container">
+                <div class="latest-news-item">
+                    <img class="latest-news-item-img" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Untitled-design21-516x270.png" alt="item 1">
+                    <div class="latest-news-item-text-container">
+                        <h3 class="latest-news-item-h3"><a class="latest-news-item-h3-a" href="#">No clue how to handle coronavirus as a startup founder? Here's what we're doing</a></h3>
+                        <h4 class="latest-news-item-h4"><a class="latest-news-item-h4-a" href="#">Patrick de Laive</a> - 4 hours ago</h4>
+                    </div>
```

Comment:
- [ ] Check the size of the articles' images in the mobile version. They should be smaller.

- [ ] Check the section's title. It should be bold and larger.

![Screenshot from 2020-03-18 16-24-52](https://user-images.githubusercontent.com/51926937/77013151-1386b580-6935-11ea-8b72-527b9279ef48.png)

Original:
![Screenshot from 2020-03-18 16-24-59](https://user-images.githubusercontent.com/51926937/77013157-17b2d300-6935-11ea-81bc-068a05211a1d.png)

- [ ] Check the tablet version. The articles should display three in a row. This happens around 990px screen size, so take a look at your media queries px. Yours are like:
![Screenshot from 2020-03-18 16-26-06](https://user-images.githubusercontent.com/51926937/77013254-4cbf2580-6935-11ea-8d43-abe7b76eab0e.png)

Should be like:
![Screenshot from 2020-03-18 16-26-14](https://user-images.githubusercontent.com/51926937/77013272-5183d980-6935-11ea-97a2-595fb6c52e59.png)